moonspeak/frequency/Dockerfile

```
@@ -14,13 +14,16 @@ RUN export DEBIAN_FRONTEND=noninteractive \
 RUN apt-get update && apt-get install -y tesseract-ocr
 RUN apt-get install -y tesseract-ocr-jpn
 
-COPY /frequency/requirements.txt ./
+COPY /frequencyapp/requirements.txt ./
```

Хз как работало, но надо без корневого слеша ))
```suggestion
COPY frequencyapp/requirements.txt ./
```